menus/tree-view.cson

```diff
@@ -39,6 +39,7 @@
     {'label': 'Split Right', 'command': 'tree-view:open-selected-entry-right'}
     {'type': 'separator'}
 
+    {'label': 'Add Root Folder', 'command': 'tree-view:add-root-folder'}
     {'label': 'Copy Full Path', 'command': 'tree-view:copy-full-path'}
```

Comment:
Yeah, I like your idea of having "add folder" mean "add a new root folder" and "new folder" mean "create a new folder in this location".